sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/samples/Sample1_AddAndRemoveRoleAssignment.cs

```diff
@@ -23,36 +23,24 @@ public void AddAndRemoveRoleAssignmentSync()
             // Environment variable with the Synapse workspace endpoint.
             string workspaceUrl = TestEnvironment.WorkspaceUrl;
 
-            #region Snippet:AccessControlSample1AccessControlClient
             AccessControlClient client = new AccessControlClient(new Uri(workspaceUrl), new DefaultAzureCredential());
-            #endregion
 
-            #region Snippet:AccessControlSample1GetWorkspaceAdminRole
             SynapseRole role = client.GetRoleDefinitions().Single(role => role.Name == "Workspace Admin");
-            #endregion
 
-            #region Snippet:AccessControlSample1AddRoleAssignment
             string principalId = Guid.NewGuid().ToString();
             RoleAssignmentOptions request = new RoleAssignmentOptions(roleId:role.Id, principalId:principalId);
             RoleAssignmentDetails roleAssignmentAdded = client.CreateRoleAssignment(request);
```

Comment:
Note that in our samples, we want to avoid doing implicit cast (although it is great) becuase it confuses our users.
```
Response<RoleAssignmentDetails> response =  client.CreateRoleAssignment(request);
RoleAssignmentDetails roleAssignmentAdded = response.Value;
```